SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+We aim to keep Timescale safe for everyone. 
+Publicly disclosing bugs in a public forum puts everyone in the Timescale community at risk.
+
+## Supported Versions
+
+The supported version is always the latest major release available in our repository.
+We also release regular minor versions with fixes and corrections alongside some new features as well as patchfix releases, that you should keep upgrading to.
```

Comment:
```suggestion
We also release regular minor versions with fixes and corrections alongside some new features as well as patchfix releases, that you should keep upgrading to.
Vulnerability fixes are made available as part of these patchfix releases.
```

Reply:
Instead of "patchfix" release, consider using "patch" release following the semver.org format.